Fix: #95. Remove traffic-light analogy. (#111)
ietf-wg-httpapi/ratelimit-headers: draft-ietf-httpapi-ratelimit-headers.md

Clients MUST NOT assume that a positive RateLimit-Remaining field value is a guarantee that further requests will be served.

> A low RateLimit-Remaining field value related to
> the number of requests issued in the time window
> or to the request throughput,
> indicates that the server might suddenly
> throttle out the client (see {{providing-ratelimit-fields}}).

Reviewer: Maybe 

```suggestion
When the value of RateLimit-Remaining is low, it indicates that the server may soon throttle the client (see {{providing-ratelimit-fields}}).
```
